tests/utils/scribereader_test.py

```diff
@@ -3,12 +3,83 @@
 
 import pytest
 
+import tron.utils.scribereader
 from tron.utils.scribereader import read_log_stream_for_action_run
 
 try:
     import scribereader  # noqa: F401
+    from clog.readers import S3LogsReader  # noqa: F401
 except ImportError:
-    pytest.skip("scribereader not available, skipping tests", allow_module_level=True)
+    pytest.skip("yelp logs readers not available, skipping tests", allow_module_level=True)
+
+
+def static_conf_patch(args={}):
```

Comment:
(might also be worth a comment here since i'm not 100% sure why this is necessary atm)

Reply:
Updated, thanks. Primarily adding this because without explicit patching of specific staticconf values and instead using `return_value=` in `unittest.mock.patch` it causes side effects when `staticconf.read` is called more than once, as in case with addition of new reader functionality.